aten/src/ATen/native/ReflectionPad.cpp

```diff
@@ -538,6 +596,11 @@ Tensor reflection_pad1d_cpu(const Tensor& input, IntArrayRef padding) {
   return output;
 }
 
+TORCH_IMPL_FUNC(reflection_pad1d_out_cpu)
+(const Tensor& input, IntArrayRef padding, const Tensor& output) {
+  reflection_pad1d_out_template(output, input, padding);
+}
```

Comment:
Ditto here

Reply:
reflection_pad1d_out_template is still used in reflection_pad1d_cpu for reflection_pad1d to dispatch QuantizedCPU.